src/Runner.Worker/ActionManager.cs

```diff
@@ -395,6 +395,11 @@ public Definition LoadAction(IExecutionContext executionContext, Pipelines.Actio
                             Trace.Info($"Action cleanup plugin: {plugin.PluginTypeName}.");
                         }
                     }
+                    else if (definition.Data.Execution.ExecutionType == ActionExecutionType.Composite && !String.IsNullOrEmpty(Environment.GetEnvironmentVariable("TESTING_COMPOSITE_ACTIONS_ALPHA")))
+                    {
+                        var compositeAction = definition.Data.Execution as CompositeActionExecutionData;
+                        Trace.Info($"Action steps: {compositeAction.Steps}.");
```

Comment:
I think this will print `Action steps: Object`

Reply:
If the json representation is desired (too noisy? esp with large inline scripts?) then see StringUtil or IOUtil (has function to convert to/from json)

Reply:
@TingluoHuang thoughts?

Reply:
Also, `compositeAction.Steps` is of type `List<Pipelines.ActionStep>`. If we were to output this, what would be the best way to output the values in a json string format?

Right now, this Trace.info statement returns the object string representation: `GitHub.DistributedTask.Pipelines.ActionStep`

If we were to print out the steps, it might make more sense to print them out in the `LoadCompositeSteps` function

Reply:
From @TingluoHuang (thanks Ting!!) : 

> Verbose will shows up only in dev mode (you build the runner from source code.)
> Trace.Info($"Load {compositeAction.Steps.Count} action steps.");
> Trace.Verbose($"Details: {StringUtil.ConvertToJson(compositeAction.Steps}");

So, I'll do that.